IHP/IDE/SchemaDesigner/Types.hs

```diff
@@ -34,8 +34,8 @@ data Statement
     | CreateIndex { indexName :: Text, unique :: Bool, tableName :: Text, expressions :: [Expression], whereClause :: Maybe Expression, indexType :: Maybe IndexType }
     -- | DROP INDEX indexName;
     | DropIndex { indexName :: Text }
-    -- | CREATE OR REPLACE FUNCTION functionName() RETURNS TRIGGER AS $$functionBody$$ language plpgsql;
-    | CreateFunction { functionName :: Text, functionBody :: Text, orReplace :: Bool, returns :: PostgresType, language :: Text }
+    -- | CREATE OR REPLACE FUNCTION functionName(param1 TEXT, param2 INT) RETURNS TRIGGER AS $$functionBody$$ language plpgsql;
+    | CreateFunction { functionName :: Text, functionParameters :: [(Text, Text)], functionBody :: Text, orReplace :: Bool, returns :: PostgresType, language :: Text }
```

Comment:
Postgres itself refers to these as `arguments` instead of `parameters` (e.g. `argname` or `argtype` in the docs). Could we rename this from parameters to arguments to be consistent with postgres?